enhancements/storage/csi-driver-install.md

```diff
@@ -193,7 +231,10 @@ Bases on current upstream plans & assumptions.
 * All sidecars and openshift/origin should run tests with forked OCP sidecars and forked OCP driver.
 
 ### OCP-4.6 (Kubernetes 1.19)
-* Add support for Azure and Cinder CSI drivers.
+* AWS EBS driver becomes mandatory, installed in all clusters. User can use both drivers at the same time. The default storage class is for in-tree volume plugin (unless cluster admin changes it).
```

Comment:
what does 'mandatory' mean?

Reply:
Installed by default (TODO: replace this tomorrow).

Reply:
replaced "mandatory" with "installed by default" in most places.